modules/Microsoft.VirtualMachineImages/imageTemplates/.test/common/deploy.test.bicep

```diff
@@ -58,35 +49,27 @@ module testDeployment '../../deploy.bicep' = {
     name: '<<namePrefix>>${serviceShort}001'
     customizationSteps: [
       {
-        restartTimeout: '30m'
+        restartTimeout: '15m'
         type: 'WindowsRestart'
       }
     ]
     imageSource: {
       offer: 'Windows-10'
       publisher: 'MicrosoftWindowsDesktop'
-      sku: '19h2-evd'
+      sku: 'win10-22h2-avd'
       type: 'PlatformImage'
       version: 'latest'
     }
-    userMsiName: resourceGroupResources.outputs.managedIdentityName
-    buildTimeoutInMinutes: 0
+    buildTimeoutInMinutes: 120
     imageReplicationRegions: []
-    lock: 'CanNotDelete'
-    managedImageName: '<<namePrefix>>-mi-${serviceShort}-001'
+    lock: ''
+    managedImageName: ''
     osDiskSizeGB: 127
-    roleAssignments: [
-      {
-        roleDefinitionIdOrName: 'Reader'
-        principalIds: [
-          resourceGroupResources.outputs.managedIdentityPrincipalId
-        ]
-        principalType: 'ServicePrincipal'
-      }
-    ]
+    roleAssignments: []
     sigImageDefinitionId: resourceGroupResources.outputs.sigImageDefinitionId
     subnetId: ''
-    unManagedImageName: '<<namePrefix>>-umi-${serviceShort}-001'
+    unManagedImageName: ''
```

Comment:
Any reason for this change?

Reply:
I think we have it here to export multiple image types

Reply:
See other comment

Reply:
combined tests. all parameters tested in Common.